src/Compilers/Core/Portable/PEWriter/TypeReferenceIndexer.cs

```diff
@@ -46,6 +49,15 @@ public override void Visit(IPlatformInvokeInformation platformInvokeInformation)
 
         protected override void ProcessMethodBody(IMethodDefinition method)
         {
+            if (method.HasBody())
```

Comment:
>if (method.HasBody()) [](start = 12, length = 21)

This looks suspicious. We didn't need to specially process method bodies for NoPia in order to detect references in them. Why do we need to process them now?

Reply:
Well, I'm really confused now because removing this doesn't fail any tests, but I was sure I only added it after I ran into failures :-(
Good catch. Thanks

---
In reply to: [215089974](https://github.com/dotnet/roslyn/pull/29180#discussion_r215089974) [](ancestors = 215089974)